cmd/backup-manager/app/backup/backup.go

```diff
@@ -40,16 +40,18 @@ func (bo *Options) backupData(backup *v1alpha1.Backup) error {
 	if backup.Spec.BR.ClusterNamespace == "" {
 		clusterNamespace = backup.Namespace
 	}
-	args, err := constructOptions(backup)
-	if err != nil {
-		return err
-	}
+	args := make([]string, 0)
 	args = append(args, fmt.Sprintf("--pd=%s-pd.%s:2379", backup.Spec.BR.Cluster, clusterNamespace))
 	if bo.TLSCluster {
 		args = append(args, fmt.Sprintf("--ca=%s", path.Join(util.ClusterClientTLSPath, corev1.ServiceAccountRootCAKey)))
 		args = append(args, fmt.Sprintf("--cert=%s", path.Join(util.ClusterClientTLSPath, corev1.TLSCertKey)))
 		args = append(args, fmt.Sprintf("--key=%s", path.Join(util.ClusterClientTLSPath, corev1.TLSPrivateKeyKey)))
 	}
+	newArgs, err := constructOptions(backup)
```

Comment:
add some comment on why this should put after the args we set internal.
better to avoid this

Reply:
added in https://github.com/pingcap/tidb-operator/pull/3360/commits/a3ceeeeee3948cb7ac06510262d412a71d0a924b